Umweltzone/src/main/java/de/avpptr/umweltzone/zones/ZonesAdapter.kt

```diff
@@ -49,16 +50,12 @@ class ZonesAdapter(
                 onItemViewInflationError.invoke(viewType)
                 error("Unknown view type: $viewType")
             }
-        }
+        } as ZoneViewHolder<ZoneViewModel>
```

Comment:
I played around with a similar idea but I could not get it working as I wanted. In the end this cast is as reasonable as the cast I did in `onBindViewHolder` on the `viewModel`s. :+1: